pkg/cli/credential/azure_credential_management.go

```diff
@@ -125,7 +125,7 @@ func (cpm *AzureCredentialManagementClient) Get(ctx context.Context, credentialN
 		return ProviderCredentialConfiguration{}, err
 	}
 
-	azureCredential, ok := resp.AzureCredentialResource.Properties.(*ucp.AzureCredentialProperties)
+	azureCredential, ok := resp.AzureCredentialResource.Properties.(ucp.AzureCredentialPropertiesClassification)
```

Comment:
doesn't look like it (this is the coverage report:
<img width="821" alt="image" src="https://github.com/radius-project/radius/assets/32342549/c2dddd5b-602f-48d1-895a-ff553a97a21d">

this code is tested in the ucp cloud functional tests. since the PR introduced breaking changes for the functional tests this failure was missed.

Reply:
Should we add a unit test then?

Reply:
yes, looking into it now